srcpkgs/logisim-evolution/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'logisim-evolution'
+pkgname=logisim-evolution
+version=3.8.0
+revision=1
+build_style=fetch
+depends="openjdk17-jre"
+short_desc="Digital logic design tool and simulator"
+maintainer="voidbert <humbertogilgomes@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/logisim-evolution/logisim-evolution"
+distfiles="
+ https://github.com/logisim-evolution/logisim-evolution/releases/download/v${version}/logisim-evolution-${version}-all.jar
+ https://raw.githubusercontent.com/logisim-evolution/logisim-evolution/v${version}/artwork/logisim-evolution-icon.svg"
```

Comment:
we should be building this ourselves, not using prebuilt artifacts

Reply:
Thank you for the guidance! I am still an xbps packaging newbie. I'll address these issues and update the PR.